### ISSUE TITLE
Table layout causes problems for screen magnifier users

### ISSUE THREAD

**Issue by accessiblewebuk** [alphagov/govuk_publishing_components#604]
Example:
![screen shot 2017-09-12 at 09 47 53](https://user-images.githubusercontent.com/18284458/30316556-86e34094-979f-11e7-9ee4-da02cb095ee3.png)

The wide spacing between the end of the data in the first column and the start of the second column means that anyone using magnification at reasonably high levels has to traverse across a lot of white space. This makes it difficult to associate the data together. Tables shouldn't need to expand to the full width and should have minimal spacing between columns. 

Believe the code is https://github.com/alphagov/static/blob/f27d6aa124bfe22cc87f960a435311c1e2bdf4a8/app/assets/stylesheets/govuk-component/govspeak/_tables.scss - presumably     width: $full-width; is what makes this happen?

**Comment by alex-ju** [comment]
Reviewing issues on this repo. The tables don't use the mentioned styles from `static` anymore and they scale to content (as in the screen capture below). This is the case for all tables published on GOV.UK using `govspeak` so I think the issue is not there anymore @accessiblewebuk – but if you have any further concerns let me know and I'll reopen it.

<img width="1150" alt="Screenshot 2021-05-13 at 16 13 56" src="https://user-images.githubusercontent.com/788096/118146333-67d8d880-b406-11eb-821b-a84bde3b4e2b.png">
